packages/data-store-typeorm/src/migration/index.ts

```diff
@@ -18,7 +19,7 @@ const migrations = [
 
 export async function migrate({dataStore}: ContextProps) {
   // bootstrap v2 data
-  await bootstrapTables(dataStore.db);
+  await bootstrapTables(dataStore.db as any as DataSource);
```

Comment:
why not use the getDataSource() helper here?

Reply:
good point, I've pushed an update